### ISSUE TITLE
Allow fields formatting

### ISSUE THREAD

**Issue by Havock94** [Budibase/budibase#5129]
**Describe the feature request**
It would be nice to be able to define formats for fields like dates and numbers, where every localization has its own "rules" for displaying values.

E.g. for an application with Italian locale, I would like to be able to format a date as DD/MM/YYYY or format a number to always have 2 decimal digits, a thousand separator (.) and a decimal separator (,) (and maybe a prefix/suffix symbol like €, but this can be addressed in the label for now)

**Comment by agruenberg** [comment]
In my opinion this is a very important feature for application development. Nearly any application needs number and date formatting.

**Comment by morphace** [comment]
Yes, can't believe that this is not possible at the moment. Extremely important feature.

**Comment by melohagan** [comment]
There is a custom date picker component for using specific locales if it helps here: https://github.com/melohagan/budibase-component-kasper-date-picker

It supports Italian among a few others.

**Comment by Havock94** [comment]
@melohagan thank you, do you have any docs on how to include it in apps? (I only see the standard date picker when adding form fields).

Thank you!

**Comment by melohagan** [comment]
> @melohagan thank you, do you have any docs on how to include it in apps? (I only see the standard date picker when adding form fields).
> 
> Thank you!

I should have linked that - here you go! https://docs.budibase.com/docs/custom-plugin#importing-plugins
